source/usage-examples/find.txt

```diff
@@ -4,13 +4,16 @@ Find Multiple Documents
 
 .. default-domain:: mongodb
 
-You can query for multiple documents in a collection with
-``collection.find()``. The ``find()`` method uses a query document that you
-provide to match the subset of the documents in the collection that match the
-query. If you don't provide a query document (or if you provide an empty
-document), MongoDB returns all documents in the collection. For more
-information on querying MongoDB, see our
-:doc:`documentation on query documents </fundamentals/crud/query-document>`.
+You can query for multiple documents in a collection by passing a
+query document to the ``collection.find()`` method. A query document
```

Comment:
```suggestion
You can query for multiple documents in a collection with the ``collection.find()`` method. A query document
```

Issue: I think was worded correctly before. As I understand it you are not required to pass a document, so I believe it is not correct to say that "passing a document causes the query to occur", so I think this should be reverted to the original wording.

Applies to both instances.